index.html

```diff
@@ -42,7 +42,7 @@
           <a class="desktop-link" href="#">Contact</a>
         </li>
         <li>
-          <a href="#"><img src="assets/icon-mail.svg" alt="mail" /></a>
+          <a href="#"><img src="assets/icon-mail.svg" alt="sent me an email" /></a>
```

Comment:
- [ ] You have done a great job implementing accessibility in your portfolio :muscle: but there is an issue with the `alt` attributes. Remember that these should be understandable when the image is not showing and the text "send me an email" is almost as using supporting text in the UI, and this can be confusing. I would suggest being more descriptive with these values (e.g., Email icon), so the user knows it is an icon and not only a text. :wink: